apps/web/app/(landing)/page.tsx

```diff
@@ -12,25 +12,35 @@ import { Github } from "lucide-react";
 import Link from "next/link";
 export const revalidate = 60;
 
-const [workspaces, apis, keys, totalVerifications] = await Promise.all([
-  db
-    .select({ count: sql<number>`count(*)` })
-    .from(schema.workspaces)
-    .then((res) => res.at(0)?.count ?? 0),
-  db
-    .select({ count: sql<number>`count(*)` })
-    .from(schema.apis)
-    .then((res) => res.at(0)?.count ?? 0),
-  db
-    .select({ count: sql<number>`count(*)` })
-    .from(schema.keys)
-    .then((res) => res.at(0)?.count ?? 0),
-  getTotalVerifications({}).then((res) => {
-    return res.data.reduce((acc, curr) => acc + curr.verifications, 0);
-  }),
-]);
+async function NumbersServed() {
+  let workspaces = 0;
+  let apis = 0;
+  let keys = 0;
+  let totalVerifications = 0;
 
-function NumbersServed() {
+  try {
+    const [workspaces_, apis_, keys_, totalVerifications_] = await Promise.all([
+      db
+        .select({ count: sql<number>`count(*)` })
+        .from(schema.workspaces)
+        .then((res) => res.at(0)?.count ?? 0),
+      db
+        .select({ count: sql<number>`count(*)` })
+        .from(schema.apis)
+        .then((res) => res.at(0)?.count ?? 0),
+      db
+        .select({ count: sql<number>`count(*)` })
+        .from(schema.keys)
+        .then((res) => res.at(0)?.count ?? 0),
+      getTotalVerifications({}).then((res) => {
+        return res.data.reduce((acc, curr) => acc + curr.verifications, 0);
+      }),
+    ]);
```

Comment:
I think using `Promise.allSettled` might be the best thing here. This will allow at the very least showing the values that can be retrieved. Also think it might be potentially best bet to completely omit any stats that do load rather than defaulting to showing 0.